### ISSUE TITLE
[WCAG 2.2. alignment] - Compliance-related work (ambulatory, low-vision)

### ISSUE THREAD

**Issue by trallard** [Quansight-Labs/jupyter-a11y-mgmt#11]

**Comment by trallard** [comment]
This should be tracked by gh-12 moving forward. Closing now
